module2/exercises/back_button_test.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>First Mobile Example</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+
+</head>
+
+<body>
+    <!-- Benjamin Cook
+     assignment?.html
+     INFO2439
+     Thoendel
+     9/4/2019
+-->
+    <div data-role="page">
+        <div data-role="header">
+            <h1>My Header</h1>
+        </div>
+        <div data-role="content">
+            <p>
+                <a href="#subpage">Go to the sub page...</a>
+            </p>
+        </div>
```

Comment:
These are the dynamic back button approaches I had hoped to find in your assignment.